src/python/ensembl/production/hive/ensembl_genome_metadata/MetadataUpdaterHiveProcessDb.py

```diff
@@ -0,0 +1,69 @@
+#  See the NOTICE file distributed with this work for additional information
+#  regarding copyright ownership.
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from pathlib import Path
+import re
+import json
+from ensembl.production.hive.BaseProdRunnable import BaseProdRunnable
+from sqlalchemy.engine.url import make_url
+
+class MetadataUpdaterHiveProcessDb(BaseProdRunnable):
+    def fetch_input(self):
+        # input_data = self.get_input_data()
+        # self.param("input_data", input_data)
+        # self.param("metadata_uri", input_data.get("metadata_uri"))
+        # self.param("database_uri", input_data.get("database_uri"))
+        # self.param("e_release", input_data.get("e_release"))
+        # self.param("email", input_data.get("email"))
+        # self.param("timestamp", input_data.get("timestamp"))
+        # self.param("comment", input_data.get("comment"))
+        # self.param("payload", input_data.get("payload"))
+        if self.param('payload'):
+            payload = json.loads(self.param('payload'))
+            self.param("comment", payload["input"]["comment"])
+            self.param("database_uri", payload["input"]["database_uri"])
+            self.param("email", payload["input"]["email"])
+            self.param("metadata_uri", payload["input"]["metadata_uri"])
+            self.param("source", payload["input"]["source"])
+            self.param("timestamp", payload["input"]["timestamp"])
```

Comment:
Raise an error if not `payload` passed

Reply:
Sorry I wasn't clear in my previous message, I guess. 
1. I believe you can call a method called "seld.param_required" if I remember well https://github.com/Ensembl/ensembl-hive/blob/2558e67ec603f0fe706a22143d1f953baa001d7b/wrappers/python3/eHive/process.py#L234 
2. I actually meant to raise an error within pipeline (something like JobFailedException : https://github.com/Ensembl/ensembl-hive/blob/2558e67ec603f0fe706a22143d1f953baa001d7b/wrappers/python3/eHive/process.py#L41) if the json.loads fail with a wrong JSON
sorry for the misunderstanding.

Reply:
Ok. So I simplified the method. I removed the if statement and replaced it with the param_required. I then added the param required to the db parameters, as these are the only essential parameters. It should throw a proper error now.